src/Sulu/Bundle/MediaBundle/Controller/MediaStreamController.php

```diff
@@ -120,10 +119,12 @@ protected function getFileResponse($fileVersion, $dispositionType = ResponseHead
             fclose($handle);
         });
 
+        $pathInfo = pathinfo($fileName);
+
         // Prepare headers
         $disposition = $response->headers->makeDisposition(
             $dispositionType,
-            basename($fileName)
+            preg_replace('/[^A-Za-z0-9\-]/', '', $pathInfo['filename']) . '.' . $pathInfo['extension']
```

Comment:
@wachterjohannes  don't overwrite the second parameter
set the third parameter with the regex!